addons/website_sale/static/src/snippets/s_add_to_cart/000.scss

```diff
@@ -0,0 +1,6 @@
+a:disabled {
```

Comment:
I think this file is not necessary.

Moreover it will apply to all `a` disabled tags on the website if the file is loaded which is not wanted.